src/lib.rs

```diff
@@ -122,6 +121,8 @@ use std::str::FromStr;
 
 use unicode_width::UnicodeWidthStr;
 
+mod tests;
```

Comment:
We can add an attribute to this `mod tests` so that it's only compiled when running `cargo test` like this:

```rust
#[cfg(test)]
mod tests;
```

Reply:
Thanks for your review. Address comments.